net/pcap-mode.el/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        orgcandman pcap-mode 52780669af0ade136f84d73f21b4dbb7ab655416
+name                ${github.project}.el
+# version is commit date
+version             2016.10.25
+categories-append   net editors
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     noarch
+license             GPL-2+
+fetch.type          git
```

Comment:
this doesn't seem to be necessary - can you please add `github.tarball_from archive`, and add the `checksums` instead as we normally prefer?

Reply:
Done, thanks for the correction @reneeotten.